src/mcts/node.cc

```diff
@@ -338,14 +338,16 @@ V4TrainingData Node::GetV4TrainingData(GameResult game_result,
   // Populate probabilities.
   const float total_n = static_cast<float>(GetChildrenVisits());
```

Comment:
`total_n` is actually an int, and it's only used once as `float` (as denominator in line 350).
Do you think it would be more readable to have it int (or `auto`) here, and do `static_cast` at the point of usage in line 350 instead?

Reply:
Indeed, and this also revealed that `GetChildrenVisits()` returns uint32_t, so the test can be simplified.